### DISCUSSION TITLE
Rules Engine in MS products/solutions

### DISCUSSION THREAD

**Discussion by as-ghub** [microsoft/RulesEngine#223] [General]
Could you please let me know some of the Microsoft products / solutions that this rules engine is being used by behind the scenes?

**Comment by TWolverson** [comment]
I would also be keen to know; in particular whether this is a greenfield development or has ancestry in other products. Most notably the term "Rules Engine" lights up some things in the mind of a (former?) BizTalk developer. A migration path from other rules-like implementations that could be enabled by knowing this, could be really helpful.
